tests/pw/.env.example

```
@@ -1,51 +1,41 @@
-DB_PREFIX=wp                                                                    [Prefix for database tables]
\ No newline at end of file
+# Dokan Configuration
+DOKAN_PRO=true                              # Set to false for Lite only
+LICENSE_KEY=your_dokan_pro_license_key      # Required for Pro
+GMAP=your_google_maps_api_key               # Optional
+
+# Playwright Configuration
+BASE_URL=http://localhost:9999
+HEADLESS=false
+CI=true
+NO_SETUP=false                             # MUST be true for Docker
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)